lib/dest/index.js

```diff
@@ -14,8 +15,9 @@ function dest(outFolder, opt) {
 
   var saveStream = pumpify.obj(
     prepare.dest(outFolder, opt),
+    makeDirs(opt),
```

Comment:
There's likely many tests that need to be written around sourcemaps because I don't trust it's fully stable yet (as signified by the <1.0 version)